### PR TITLE
gives many things proper recipes, and other fixes

### PR DESCRIPTION
**Pull request by terezi4real**
## About the PR

Changes placeholder recipes to actual gameplay ones, more work to be done in the future.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: fixes recipes

### CODE REVIEW

**Comment by terezi4real** [comment]
i fucked somethin up

File: Resources/Prototypes/Research/arsenal.yml
Comment (sTiKyt) [inline Resources/Prototypes/Research/arsenal.yml:1]: Why are you commenting out researches?

File: Resources/Prototypes/Research/disciplines.yml
Comment (sTiKyt) [inline Resources/Prototypes/Research/disciplines.yml:1]: Same question here

File: Resources/Prototypes/_RedEclipse/Entities/Objects/Factory/Structures/Assemblers/FactoryClownAssembler.yml
Comment (sTiKyt) [inline Resources/Prototypes/_RedEclipse/Entities/Objects/Factory/Structures/Assemblers/FactoryClownAssembler.yml:1]: Dynamic recipes are the ones unlocked through research?

File: Resources/Prototypes/_RedEclipse/Recipes/Factory/Assemblers/assembler_medical.yml [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
   materials: # TODO better recipe
-    FactoryBasicParts : 1
+    FactoryBasicCasings : 1
+    FactoryInjectors: 1
+    FactoryMedicalParts: 1
 
 - type: latheRecipe
-  id: FactorySyndicateSmokesPacket
-  result: CigPackSyndicate
-  completetime: 1
```
Comment (sTiKyt) [inline Resources/Prototypes/_RedEclipse/Recipes/Factory/Assemblers/assembler_medical.yml:33]: Well, if you remove FactorySyndicateSmokesPacket proper commit would be "Removed FactorySyndicateSmokesPacket recipe", bro, it's impossible to figure all this out properly fast when proper commits or changelog are non-existent

**Review by sTiKyt — CHANGES_REQUESTED** [review]
Changelog request:

- [ ] Specify the items that were removed.
- [ ] Specify the items that were added.
- [ ] Specify the items that now have a proper recipe.
- [ ] Specify the items that are now crafted in a different machine.


That's it for now. I will review further after this is resolved.

**Pull request closed without merging**